test/src/test/java/hudson/model/NodeTest.java

```diff
@@ -237,6 +242,26 @@ public void testToComputer() throws Exception {
         assertNull("Slave which is not added into Jenkins list nodes should not have assigned computer.", node.toComputer());
         assertNotNull("Slave which is added into Jenkins list nodes should have assigned computer.", slave.toComputer());
     }
+
+    @Issue("JENKINS-26391")
+    @Test
+    public void testGetAssignedLabelWithJobs() throws Exception {
+        final Node node = j.createOnlineSlave();
+        node.setLabelString("label1 label2");
```

Comment:
Added several new test units one of which includes a more complex label expression.